ldms/src/core/ldms_xprt.h

```diff
@@ -391,8 +391,7 @@ struct ldms_xprt {
 
 	int term;
 
-	/** Transport message logging callback */
-	ldms_log_fn_t log;
+//	ldms_log_fn_t log;
```

Comment:
Please remove, let's not leave commented-out code in the tree.

Reply:
Done! Thanks for catching this.